src/controllers/funds.js

```diff
@@ -0,0 +1,32 @@
+/* eslint-disable consistent-return */
+const mongoose = require('mongoose');
+const Funds = require('../models/fund');
+
+module.exports = {
```

Comment:
please follow up with the team and make sure everyone follows the same convention of exporting functions